sql/sys_vars.cc

```diff
@@ -7625,6 +7626,24 @@ static Sys_var_ulonglong Sys_tf_sequence_table_max_upper_bound(
     GLOBAL_VAR(tf_sequence_table_max_upper_bound), CMD_LINE(REQUIRED_ARG),
     VALID_RANGE(1024, ULLONG_MAX), DEFAULT(1048576), BLOCK_SIZE(1));
 
+static Sys_var_charptr Sys_var_buffered_log_filename(
+    "buffered_log_filename", "Filename for buffered log",
+    GLOBAL_VAR(buffered_log_filename), CMD_LINE(REQUIRED_ARG), IN_FS_CHARSET,
+    DEFAULT("/tmp/buffered_log.log"));
+
+std::size_t buffered_log_size;
+
+static bool buffered_log_size_update(sys_var *, THD *, enum_var_type) {
+  buffered_log.resize(buffered_log_size);
+  return false;
+}
+
+static Sys_var_ulonglong Sys_var_buffered_log_size(
+    "buffered_log_size", "Size of the buffered log",
```

Comment:
buffered_error_log_size

Also please change all variable name to indicate error or err